apps/data-types.md

```diff
@@ -577,6 +577,8 @@ This object is used for the attributes of basePrice, depositPrice, ecoParticipat
 | sfUrl | string | The link to the storefront URL of the product. |
 | productNumber | string | The product number. |
 | isProductNumberVisible | boolean | Indicates if the product number is displayed in the storefront. |
+| isManufacturerPriceRRP | boolean | Indicates if the manufacturer price is to be treated as an RRP. |
+| isShippingLinkVisible | boolean | Indicates if the shipping link is displayed on the VAT text. |
```

Comment:
```suggestion
| isShippingLinkVisible | boolean | Indicates if a shipping cost reference that links to the shop's payment & shipping page is displayed together with the product price in the storefront. |
```
I think it would be helpful to give developers more detailed information about where the shipping link leads to and that the attribute refers to the storefront. In addition I would suggest to leave out the information about the VAT. If I'm not mistaken the tax info is handled by another attribute. As the VAT info is not necessarily displayed (no Vat products) we would need to describe that case as well.
Furthermore, I think this attribute needs to be added to ##productSearch as well, but I'm not sure. Maybe I'm totally wrong about that one 😁

Reply:
I think your suggestion is the best way to describe that attribute. It's now clear where the link leads to.
For the productSearch the attribute is already in place: /_raml/apps/search/examples/post_search_products_response.json

Reply:
@mhoehn Yes, it is added to the example but what I meant was to put in data-types.md under ##productSearch.
![Bildschirm­foto 2023-03-03 um 15 55 50](https://user-images.githubusercontent.com/108728944/222752761-a3602580-6232-4960-b149-972900dadd08.png)